src/Raven.Server/Documents/Indexes/Errors/FaultyInMemoryIndex.cs

```diff
@@ -18,12 +18,12 @@
 using Raven.Server.Documents.Queries.Timings;
 using Raven.Server.ServerWide;
 using Raven.Server.ServerWide.Context;
-
+using Voron.Impl;
```

Comment:
Is it needed?

Reply:
Nope, leftover from when I removed the code that was there when applying the review ideas.